portfolio/src/main/webapp/script.js

```diff
@@ -2,42 +2,64 @@
 
 let elementsToShow;
 let scroll;
+let quotes =
+    ["\"In a world too often governed by corruption and arrogance, it can be difficult to stay true to one's philosophical and literary principles.\"   -Lemony Snicket, A Series of Unfortunate Events", "\"Did you know that the Sparrow flies south for winter?\"   -Skulduggery Pleasant, Skulduggery Pleasant", "\"Someday in your life, you'll realize there is no turning back. There was never any turning back.\"   -u/JayStar1213"];
+let entrances = 
+    ["animate__bounceInUp", "animate__bounceInLeft", "animate__bounceInRight", "animate__bounceInDown", "animate__fadeInBottomLeft", "animate__fadeInBottomRight"];
 
 $(document).ready(function() {
+    // scroll here is a fallback function in case some browswers don't support method requestAnimationFrame. 
     scroll = window.requestAnimationFrame || function(callback){ window.setTimeout(callback, 1000/60)};
```

Comment:
nit: consider using a conditional operator here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
changed to tertiary operator